### PR TITLE
Release Notes for May 2022

### PR DESCRIPTION
**Pull request by HBehrens**

### CODE REVIEW

File: blog/2022-05-31.mdx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+---
+
+### Memfault OTA for SWUpdate users
+
+Memfault’s over-the-air update service is now
+[available on Embedded Linux](/docs/linux/linux-releases-integration-guide) and
+SWUpdate [via the hawkBit DDI](/docs/linux/linux-memfault-hawkbit-comparison).
+This makes all Memfault OTA management and hosting features such as cohorts,
+staged rollouts, full vs. delta releases, and a scalable global CDN available to
+Linux devices that utilize one of the most-popular update agents. Memfault also
```
Comment (victorbordo) [inline blog/2022-05-31.mdx:12]:
```suggestion
Linux devices that utilize one of the most popular update agents. Memfault also
```

File: blog/2022-05-31.mdx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+added support for forced (non-interactive) updates – invaluable for delivering
+security updates to embedded IoT devices.
+
+![Memfault ❤️ Linux.png](/img/blog/2022-05-31-MemfaultHeartLinux.png)
+
+To communicate with SWUpdate clients,
+[Memfault acts as a hawkBit update server](/docs/linux/linux-memfault-hawkbit-comparison).
+It implements the Direct Device Integration API equally enabling other on-device
```
Comment (victorbordo) [inline blog/2022-05-31.mdx:20]:
```suggestion
It implements the Direct Device Integration API enabling other on-device
```

File: blog/2022-05-31.mdx [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+- Improved: User flow for Single Sign-On
+- Improved: UX when editing/deleting metrics to inform about affected charts and
+  alerts
+- Improved: Manually refreshing charts updates data for all accessible time
+  ranges
+- Improved: UI for Issues reflects data retention of raw data (e.g. “Recent
+  Traces”)
+- Improved: Alerts can only be configured with compatible metrics (time-series
+  vs. attributes)
+- Improved: New category “DangerZone” in settings for irreversible operations on
```
Comment (victorbordo) [inline blog/2022-05-31.mdx:37]:
```suggestion
- Improved: New category “Danger Zone” in settings for irreversible operations on
```

File: blog/2022-05-31.mdx [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+- Improved: Various UI elements (app-level status badge, column header on issues
+  list, removed “Beta 🧪” badge from several places, tooltip for charts, title
+  on release details page, clarified grouping on cohort’s version distribution,
+  tooltips in version matrix)
+- Fixed: UI glitch when changing custom metric filter in device search
+- Fixed: Various UI glitches in issue details page (responsivenes, resetting
+  search criteria)
+- Fixed: Multi-select on Issue list broken
+- Fixed: Timeout for page Settings→Hardware for some projects
+
```
Comment (victorbordo) [inline blog/2022-05-31.mdx:48]:
```suggestion
Fixed: Issue where device attributes initially created with a Never write policy could not be updated after changing the policy to Device-Writable
```

File: blog/2022-05-31.mdx [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+- Improved: Filtering by logging source for Android log viewer
+- No new version of
+  the [Android Bort AOSP SDK](https://github.com/memfault/bort) in May.
+
+## Linux
+
+- Added:
+  [API endpoint for hawkBit](/docs/linux/linux-memfault-hawkbit-comparison)
+  (incl. support for forced vs. soft releases)
```
Comment (victorbordo) [inline blog/2022-05-31.mdx:76]:
```suggestion
  (including support for forced vs. soft releases)
```

**Review by victorbordo — COMMENTED** [review]
Added a few suggestions.

File: blog/2022-05-31.mdx (hunk 4 above)
Comment (HBehrens) [reply blog/2022-05-31.mdx:48]: Interesting that you pulled this up. I removed this as an explicit point after lumping multiple aspects beneath the second point about metrics and affected entities.

**Pull request merged**